### ISSUE TITLE
When I Add New User, And Use The User to view dashboard ,it has an error 

### ISSUE THREAD

**Issue by kira8565** [Graylog2/graylog2-server#2005]
### Steps to reproduce the problem

1. Add New User And assign dashboard to the user
2. change current user to the new user which add before
3. click the dashboard
4. can not view the dashboard which i assign before,and occur an error like this 
```
Unhandled rejection TypeError: Cannot read property 'set' of undefined
    at ProxyClass._getDashboardActions (webpack:///./src/components/dashboard/Dashboard.jsx?:120:70)
    at ProxyClass.proxiedMethod (webpack:///./~/react-proxy/modules/createPrototypeProxy.js?:44:30)
    at ProxyClass.eval [as _getDashboardActions] (webpack:///./~/react-hot-api/modules/makeAssimilatePrototype.js?:15:37)
    at ProxyClass.render (webpack:///./src/components/dashboard/Dashboard.jsx?:154:14)
    at ProxyClass.proxiedMethod (webpack:///./~/react-proxy/modules/createPrototypeProxy.js?:44:30)
    at ProxyClass.eval [as render] (webpack:///./~/react-hot-api/modules/makeAssimilatePrototype.js?:15:37)
    at ReactCompositeComponentMixin._renderValidatedComponentWithoutOwnerOrContext (http://localhost:8080/build/vendor.js:38634:35)
    at ReactCompositeComponentMixin._renderValidatedComponent (http://localhost:8080/build/vendor.js:38654:33)
    at wrapper [as _renderValidatedComponent] (http://localhost:8080/build/vendor.js:5007:22)
    at ReactCompositeComponentMixin.mountComponent (http://localhost:8080/build/vendor.js:38267:31)
    at wrapper [as mountComponent] (http://localhost:8080/build/vendor.js:5007:22)
    at Object.ReactReconciler.mountComponent (http://localhost:8080/build/vendor.js:6191:36)
    at ReactDOMComponent.ReactMultiChild.Mixin.mountChildren (http://localhost:8080/build/vendor.js:41719:45)
    at ReactDOMComponent.Mixin._createContentMarkup (http://localhost:8080/build/vendor.js:39393:33)
    at ReactDOMComponent.Mixin.mountComponent (http://localhost:8080/build/vendor.js:39281:30)
    at Object.ReactReconciler.mountComponent (http://localhost:8080/build/vendor.js:6191:36)
```

and I Change This code ins dashboard.jsx 
```
<MenuItem href={ApiRoutes.StartpageController.set('dashboard', this.props.dashboard.id).url}>
```
to 
```
<MenuItem href={StartpageStore.set('dashboard', this.props.dashboard.id).url}>
```

it work 

### Environment

* Graylog Version: Graylog-Master  (Beta2)
* Elasticsearch Version: 2.1.1
* MongoDB Version: 3.2
* Operating System: Mac
* Browser version: Chorm 48

**Comment by edmundoa** [comment]
Hi,

Thank you for reporting this issue. `ApiRoutes.StartpageController` is indeed undefined, although the suggested workaround doesn't really work (the page will load, but the button doesn't work as intented). Will fix it for the next beta.

**Comment by edmundoa** [comment]
This issue has been fixed in f938cdd. Unfortunately there is another issue that makes impossible for regular users to reset their startpage. There is a PR already for that, so this should work as expected once #1987 is merged.
